trunk/SUAVE/Components/Energy/Converters/Propeller.py

```diff
@@ -487,7 +487,34 @@ def spin(self,conditions):
         return thrust, torque, power, Cp, outputs , etap
 
 
-def compute_aerodynamic_forces(a_loc, a_geo, cl_sur, cd_sur, ctrl_pts, Nr, Re, Ma, alpha, tc, Na, nonuniform_freestream):
+def compute_aerodynamic_forces(a_loc, a_geo, cl_sur, cd_sur, ctrl_pts, Nr, Na, Re, Ma, alpha, tc, nonuniform_freestream):
+    """Analyzes aerodynamic forces at sectional blade locations.
+
+    Assumptions:
+    N/A
+
+    Source:
+    N/A
+
+    Inputs:
+    a_loc                      [-]
```

Comment:
I'm sorry to be nitpicky. But I wanted an english readable definition of the variables. For example: https://suave.stanford.edu/doxygen/group__Methods-Weights-Buildups-Electric__Lift__Cruise.html#gab1de5c66fb86183af013cf4cea3ded69

Reply:
Ah gotcha, I think I fixed it but let me know if you think I am still missing anything

Reply:
Haha. I miss being in the office right now, this is so easy to explain in person. This is what I meant:

        config                                           SUAVE Config Data Structure
        speed_of_sound                          Local Speed of Sound                     [m/s]
        maximumTipMach                       Allowable Tip Mach Number           [Unitless]
        disk_area_factor                         Inverse of Disk Area Efficiency       [Unitless]
        max_thrust_to_weight_ratio      Allowable Thrust to Weight Ratio    [Unitless]
        motor_efficiency                         Motor Efficiency                               [Unitless]